README.md

```diff
@@ -18,23 +18,36 @@ For the best development experience, make sure you have the PHP linter enabled i
 
 ## User Feedback
 
-> **Another total must have for PHP developers using Visual Studio Code.**  
+> **Another total must have for PHP developers using Visual Studio Code.**
 > Jan Hajek
 
-> That's the tweet I've been waiting for since the first public release of VS Code. Thanks.  
+> That's the tweet I've been waiting for since the first public release of VS Code. Thanks.
 > [Bruno Baketaric](https://twitter.com/laphblog/status/719631906598449152)
 
-> omg omg omg omg omg  
+> omg omg omg omg omg
 > [Rich Perez](https://twitter.com/imperez/status/719645661461921793)
 
-> 👌👌👌 super awesome.  
+> 👌👌👌 super awesome.
```

Comment:
Code automatically removed whitespace from line endings since that is how I have my settings. I changed it in github to fix that issue.